src/pages/Volunteer.tsx

```diff
@@ -1,189 +1,151 @@
+            <p>We meet on most Monday nights. Join us, everyone is welcome!</p>
+            <p>
+              RSVP on <a href={ExternalURL.MEETUP}>Meetup</a> to see the full
+              schedule of events and to let us know you're coming.
+            </p>
+            <p>
+              Galvanize Denver &ndash; Platte
+              <br />
+              1644 Platte St
+              <br />
+              Denver, CO 80202
+              <br />
+              6pm-9pm
+              <br />
+            </p>
+            <span className="social-links">
+              <a href={ExternalURL.MEETUP}>
+                <FontAwesomeIcon icon={faMeetup} />
               </a>
-
-              <a style={{ color: 'black' }} href={ExternalURL.GITHUB}>
-                <Icon size="large" name="github" />
+              <a href={ExternalURL.GITHUB}>
+                <FontAwesomeIcon icon={faGithub} />
               </a>
-
-              <a style={{ color: 'black' }} href={ExternalURL.FACEBOOK}>
-                <Icon size="large" name="facebook" />
+              <a href={ExternalURL.FACEBOOK}>
+                <FontAwesomeIcon icon={faFacebookF} />
               </a>
-              <div style={{ textAlign: 'center' }}>
-                <a href={ExternalURL.MEETUP}>
-                  <Button circular color="black" style={button__css}>
-                    Meetup
-                  </Button>
-                </a>
-                <a href="contact">
-                  <Button circular color="black" style={button__css}>
-                    Contact
-                  </Button>
-                </a>
-              </div>
-            </Grid.Column>
-            <Grid.Column>
-              <div
-                style={{
-                  maxWidth: '640px',
-                  marginTop: '14px',
-                  marginBottom: '14px'
-                }}
-              >
-                <div style={{ padding: '56.25% 0 0 0', position: 'relative' }}>
-                  <iframe
-                    title="Code for Denver meeting location"
-                    src={`https://www.google.com/maps/embed/v1/place?key=${process.env.REACT_APP_GOOGLE_MAPS_API_KEY}&q=Galvanize+-+Denver+(Platte)`}
-                    style={{
-                      position: 'absolute',
-                      top: 0,
-                      left: 0,
-                      width: '100%',
-                      height: '100%',
-                      border: 0
-                    }}
-                    allow="fullscreen"
-                    allowFullScreen
-                  />
-                </div>
+            </span>
+          </div>
+          <div>
+            <div className="iframe-parent">
+              <div className="iframe-container">
+                <iframe
+                  title="Code for Denver meeting location"
+                  src={`https://www.google.com/maps/embed/v1/place?key=${process.env.REACT_APP_GOOGLE_MAPS_API_KEY}&q=Galvanize+-+Denver+(Platte)`}
```

Comment:
can we update this location to workability ?

Reply:
Updated!